cmd/commands/deploy.go

```diff
@@ -208,7 +208,18 @@ func deployAction(ctx context.Context, a inngest.ActionVersion) (inngest.ActionV
 		return a, fmt.Errorf("error preparing action: %w", err)
 	}
 
-	config, err := cuedefs.FormatAction(a)
+	// XXX: Remove the dockerfile field from the runtime struct b/c the action shouldn't need this info
```

Comment:
I'm unsure of the cleanest way to do this - Some fields may be useful in step's `runtime` locally for building, etc. but are just not needed or relevant to the action when as the image is already build. Would love suggestions on this - I thought this work around could be alright for now until we sort out a better way to decide how to do this - perhaps have 2 `runtime` structs and/or convert a `Step` struct to the action here.

Reply:
Yeah, this is _definitely_ some tech debt we accrued from the prototyping stage.  We should only need to dictate that the action is of type docker for now.  The "XXX" is a great comment so that we know we need to come back and clean this up — I'm assuming we'll get to that as we near v1 and we figure out the self hosting flows too.